onebusaway-android/src/main/java/org/onebusaway/android/ui/ArrivalsListHeader.java

```diff
@@ -1470,4 +1474,20 @@ public void showProgress(boolean visibility) {
             mProgressBar.setVisibility(View.GONE);
         }
     }
+
+    private void notifyRouteFavoriteChanged(boolean isRouteSaved) {
```

Comment:
I recommend structuring control flow like what you've written in this method like so:

```
private void notifyRouteFavoriteChanged(boolean isRouteSaved) {
    string text = isRouteSaved ? R.string.route_removed_from_favorites : R.string.route_added_to_favorites;
    Snackbar.make(mView, text, Snackbar.LENGTH_SHORT).show();
}
```